### PR TITLE
Updates metadata. Makes check if station is in metadata

### PR DESCRIPTION
**Pull request by dulte**
Small changes to eeareader

### CODE REVIEW

File: src/pyaro_readers/eeareader/EEATimeseriesReader.py [hunk 1]
```diff
@@ -73,6 +78,7 @@ def __init__(
         self._set_filters(filters)
 
         self.metadata = self._read_metadata(filename)
+        self.datafile = self._read_data()
```
Comment (heikoklein) [inline src/pyaro_readers/eeareader/EEATimeseriesReader.py]: both `datafile` and `_read_data` and `data.toml` are very confusing here. It is everything but `data`=measurement of e.g. 17.35µg/m³, please find a better name. `cfg` or `setup`? And definitely don't call something for `file` which is neither a filename nor filehandle.

**Comment by dulte** [comment]
I've added the possibility to change units with a factor. Factor is added to data.toml. Might want a separate file for these factors

**Comment by dulte** [comment]
I'm not sure with the files fail due to not finding data.toml. Will have a look next week

**Review by heikoklein — CHANGES_REQUESTED** [review]
Tests not working (still draft)? Very strange choice of variable/file-names.

**Comment by dulte** [comment]
@heikoklein . Yes, right before you answered I changed it to a draft. The tests fail because they can't find data.toml, and I'm not entirely sure why... going to look at it over the weekend.

I've added a simple unit conversion field as well in data.toml. It seems to work, but I have to add some tests for that as well before the pr is ready for review. It might also be the wrong way of doing a unit change (changing it without making a new variable). So some work before the pr is ready

**Comment by dulte** [comment]
Note to self: remove unit change from data.toml, and add it as a possible argument to the reader instead. 

Or maybe it as a wrapper to pyaro, similar to the variable name change wrapper, so all readers can do unit change.

**Comment by dulte** [comment]
@heikoklein. Fixed some problems, and made some new ones. But it is now ready for review

**Review by heikoklein — APPROVED** [review]
Looks good, except for linting. You should really fix your black setup.

**Pull request merged**